core/src/main/java/io/github/tcdl/msb/api/ResponderServer.java

```diff
@@ -1,17 +1,13 @@
 package io.github.tcdl.msb.api;
 
-import io.github.tcdl.msb.api.message.payload.Payload;
-
 /**
  * {@link ResponderServer} enable user to listen on messages from the bus and executing microservice business logic.
  * Call to {@link #listen()} method will start listening on incoming messages from the bus.
  * And also it's required to implement interface {@link RequestHandler}. Implementation of this interface will be
  * business logic processed by microservice. Inside this logic we can use instance of {@link Responder} created by {@code ResponderServer}
  * for each message from bus, and can be used for sending responses back to bus.
- *
- * @param <T> expected payload type of incoming message
  */
-public interface ResponderServer<T extends Payload> {
+public interface ResponderServer {
```

Comment:
We don't really need to have a type param here. It was overlooked in the previous version.